app/node.py

```diff
@@ -0,0 +1,9 @@
+from typing import Any, Hashable
+from dataclasses import dataclass
+
+
+@dataclass
+class Node:

```

Comment:
I'd say Node must be defined inside of a Dictionary class - because it is part of its implementation